components/about.tsx

```diff
@@ -1,81 +1,101 @@
-              fill="url(#illustration-02)"
-            />
-          </svg>
-        </div>
+import React from "react";
+import { Card, CardContent, CardDescription } from "@/components/ui/card";
 
-        {/* Hero content */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
-          {/* Section header */}
-          <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Run by nang and FryingPan
-            </h1>
-            <p
-              className="mb-8 text-xl text-gray-600"
-              data-aos="fade-up"
-              data-aos-delay="200"
-            >
-              Building in Public
+const AboutComponent: React.FC = () => {
+  return (
+    <section
+      className="to-white bg-gradient-to-b from-gray-50 px-4 py-12 pt-20"
+      data-aos="fade-up"
+      data-aos-delay="200"
+    >
+      <div className="mx-auto max-w-6xl">
+        <div className="flex flex-col lg:flex-row">
+          {/* Left column */}
+          <div className="lg:w-1/2 lg:pr-8">
+            <h2 className="mb-4 text-3xl font-bold">
+              Fully <span className="text-primary-700">transparent</span>,
+              PearAI is built in public
+            </h2>
+            <p className="mb-4 text-gray-600">
```

Comment:
```suggestion
            <p className="mb-4 text-secondary-400">
```
We should use secondary colors instead of gray to have consistent styling

Reply:
Hm, we use both secondary and gray throughout the whole codebase. This can be ignored for now until we decide officially what to use.